containers/Config.ts

```diff
@@ -0,0 +1,54 @@
+import { ethers } from "ethers";
+type Network = ethers.providers.Network;
+export const config = (network: Network | null) => {
+  const infuraId =
+    process.env.REACT_APP_INFURA_ID || "d5e29c9b9a9d4116a7348113f57770a8";
+  const infuraRpc = `https://${network?.name}.infura.io/v3/${infuraId}`;
+  const apiKey = process.env.REACT_APP_ONBOARD_API_KEY
+    ? process.env.REACT_APP_ONBOARD_API_KEY
+    : "12153f55-f29e-4f11-aa07-90f10da5d778";
+  return {
+    apiKey: apiKey ? apiKey : "12153f55-f29e-4f11-aa07-90f10da5d778",
+    portisKey: process.env.REACT_APP_PORTIS_API_KEY,
+    onboardWalletSelect: {
```

Comment:
nit: I'd recommend you put the `apiKey`, `portisKey`, `onboardWalletSelect` and `wallets` into a `portisConfig` key. I think we'll want to use this container for other external SDK's in the future and the name `apiKey` will be reused a lot

Reply:
Another nit: Should these all be CAPITALIZEd variable names since they're all hardcoded values?

i.e.

API_KEY, PORTIS_KEY, ONBOARD_WALLET_SELECT...

Reply:
I have refactored this accordingly. PTAL. 

I'm not a fan of the all caps on the hardcoded vars though. Also this is a function that can return different valued based on the context. [Airbnb](https://github.com/airbnb/javascript#naming--uppercase) standard says that uppercase constants should only be used if `the programmer can trust it (and its nested properties) to never change` but this is not really the case as the config changes with network param.